test/org/sagebionetworks/bridge/services/ScheduledActivityServiceDuplicateTest.java

```diff
@@ -45,7 +45,6 @@
  * a standard time and midnight seems to work regardless of when the timestamp is).
  */
 @RunWith(MockitoJUnitRunner.class)
-@Ignore // The current code does not prevent duplicates.
 public class ScheduledActivityServiceDuplicateTest {
```

Comment:
So to rephrase, a case where the user has one persisted time zone for the events, but then asks for those tasks back in another time zone. I believe I have tests for this, though not in this duplication test. But I added changing endsOn to the MSK time zone in the @before method, and the right tasks are still returned (without duplicates).

ScheduledActivityServiceRecurringTest manipulates the time zone extensively to verify that works correctly, so I added the initial persisted time zone as a zone different from the Moscow/PST time zones used in the test, to verify that is still correct.